Makefile

```
@@ -18,6 +18,12 @@ CFLAGS ?= -O3 -I$(LVGL_DIR)/ $(WARNINGS) `pkg-config --cflags wayland-client` `p
 LDFLAGS ?= -lSDL2 -lm `pkg-config --libs wayland-client` `pkg-config --libs xkbcommon` -lpthread
 BIN = demo
 
+CONFIG_LV_USE_THORVG_INTERNAL ?=y
+
+ifeq ($(CONFIG_LV_USE_THORVG_INTERNAL), n)
```

```suggestion
ifeq ($(CONFIG_LV_USE_THORVG_EXTERNAL), y)
```

to add ThorVG only if explicitly requested.

---

> to add ThorVG only if explicitly requested.

ok , I see